app/models/patient_business_identifier.rb

```diff
@@ -8,7 +8,8 @@ class PatientBusinessIdentifier < ApplicationRecord
     bangladesh_national_id: 'bangladesh_national_id'
   }
 
-  validates :identifier, presence: true
+  validates :identifier, presence: true, if: -> { identifier_type == 'simple_bp_passport' }
```

Comment:
What happens when it doesn't match Simple BP passport NOR Bangladesh ID?

This should be `identifier_type != 'bangladesh_national_id`